OutOfSchool/OutOfSchool.DataAccess/Repository/IDepartmentRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository
+{
+    public interface IDepartmentRepository : IEntityRepository<Department>
+    {
+        /// <summary>
+        /// Checks entity elements for uniqueness.
+        /// </summary>
+        /// <param name="entity">Entity.</param>
+        /// <returns>Bool.</returns>
+        bool SameExists(Department entity);
+
```

Comment:
this method is used in 2 repositories, maybe can be used to base interface or separate interface...

Reply:
I separated the interface.